set-client/src/main/java/eu/europeana/set/client/UserSetApiClient.java

```diff
@@ -17,6 +17,8 @@
 
 public class UserSetApiClient extends BaseUserSetApi {
 
+    public static final ThreadLocal<String> token = new ThreadLocal<>();
```

Comment:
we should move this to bsase classtogether with the getter and setter methods, so that the token can be initialized in the constructor pf the base class. We should also rename the base class to BaseUserSetApiClient or AbstractUserSetApiClient